canonical-web-and-design/ubuntu.com
# Creates /telco/magama page

## Done

- Creates /telco/magama page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco/magma
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
-  Compare against [copy-doc](https://docs.google.com/document/d/1xDoW7wsw62lYYiMB5a6i4vJBzdUFjGiyXjRcnyplDGU/edit#)
- Contact form not ready yet.


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4193
